integration/spark/src/main/scala/org/apache/spark/sql/parser/CarbonSparkSqlParserUtil.scala

```diff
@@ -683,12 +683,6 @@ object CarbonSparkSqlParserUtil {
       fields: List[Field],
       tblProp: Option[List[(String, String)]]
   ): CarbonAlterTableAddColumnCommand = {
-    fields.foreach { f =>
-      if (CarbonParserUtil.isComplexType(f.dataType.get)) {
```

Comment:
I think,  still Alter add Map type is not supported. Please handle

Reply:
done